setup.py

```diff
@@ -11,7 +11,7 @@
     packages=['twisted_tools', 'twisted_tools.scripts', 'twisted_tools.test'],
     scripts=[
         'bin/force-build',
-        'bin/mkbranch',
+        'bin/make-branch',
```

Comment:
true... it was here since #4 was not merge and I needed a branch which works and can be used